deploy/crs/kieapp_rhdm_trial.yaml

```diff
@@ -2,5 +2,10 @@ apiVersion: app.kiegroup.org/v2
 kind: KieApp
 metadata:
   name: rhdm-trial
+  annotations:
+    consolekind: ConsoleYAMLSample
```

Comment:
Why is this necessary?

Reply:
It is not needed. However, doing gives us flexibility to selective apply certain CRs (ie:apply only those have this 'consoleKind'), instead of apply the CRs in the entire folder.

Reply:
The caller will be passing a yaml to create a console yaml sample. They will want it created and we shouldn't be second-guessing their call. In fact, we should use defaults if no annotations exist at all.